lib/Ix.pm

```diff
@@ -3,3 +3,22 @@ package Ix;
 # ABSTRACT: automatic HTTP interface for applications
 
 1;
+
+=head1 OVERVIEW
+
+Ix is a framework for building applications using the [JMAP](http://jmap.io/)
+protocol.  Ix provides abstractions for writing method handlers to handle
+JMAP requests containing multiple method calls.  It also provides a mapping
+layer to automatically provide JMAP's CRUD and windows query interfaces by
+mapping to a DBIx::Class schema.
```

Comment:
Oh yeah, I did that everywhere except Ix.pm, which I just copy/pasted from the existing readme because it was the last one I did, which meant that I'd already spent like, 8 hours doing this. Will fix.